samples/test/rollout.js

```diff
@@ -0,0 +1,207 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+const {assert} = require('chai');
+const cleanup = require('./clean.js');
+const {describe, before, after, it} = require('mocha');
+const {
+  GameServerConfigsServiceClient,
+  GameServerDeploymentsServiceClient,
+  RealmsServiceClient,
+} = require('@google-cloud/game-servers');
+
+const cp = require('child_process');
+const uuid = require('uuid');
+
+const execSync = cmd => cp.execSync(cmd, {encoding: 'utf-8'});
+
+describe('Game Server Rollout Test', () => {
+  const configClient = new GameServerConfigsServiceClient();
+  const deploymentClient = new GameServerDeploymentsServiceClient();
+  const realmClient = new RealmsServiceClient();
+  let deploymentId, configId, projectId, realmId, realmLocation;
+
+  before(async () => {
+    await cleanup();
+
+    projectId = await deploymentClient.getProjectId();
+    deploymentId = `test-nodejs-${uuid.v4()}`;
```

Comment:
ProjectId needs to be defined here because of the `await` thing, but the rest of these can be assigned above when you declare the variable.  I'd rather keep the assignment and declaration together if we can.

Reply:
Done.